components/Common/Button/index.stories.tsx

```diff
@@ -0,0 +1,294 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Wouldn't it be easier to use storybook controls instead of lot of stories ?

Reply:
I do agree with that, less code, easier to maintain. but the only down side is I couldn't figure out a way to display hover and focus states with that.

Reply:
for how and focus do an dedicated stories